assets/src/components/v2/normal_header.tsx

```diff
@@ -66,8 +66,15 @@ const NormalHeaderTitle: ComponentType<NormalHeaderTitleProps> = forwardRef(
     }
 
     const abbreviatedText = fullName ? text : abbreviateText(text);
+    const environmentName = getDatasetValue("environmentName");
```

Comment:
The import is missing here. Also, the return for `getDatasetValue` is `string | undefined` so the linter might complain. You can get around that with:

```suggestion
    const environmentName = getDatasetValue("environmentName") || "";
```

Reply:
Thank you! I was wondering about a good way to get around that linter error